src/features/Schedule/CreateActivityPopup/NotificationsTab/Reminder/Reminder.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslation } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+
+import { TimePicker } from 'components';
+import { InputController } from 'components/FormComponents';
+import { StyledLabelLarge } from 'styles/styledComponents/Typography';
+import theme from 'styles/theme';
+import { StyledFlexTopCenter } from 'styles/styledComponents/Flex';
+
+import { StyledReminder, StyledInputWrapper } from './Reminder.styles';
+import { Header } from '../Header';
+import { FormValues } from '../../CreateActivityPopup.types';
+import { StyledColInner, StyledNotificationWrapper } from '../NotificationsTab.styles';
+
+export const Reminder = () => {
+  const { t } = useTranslation('app');
+  const { setValue, control } = useFormContext<FormValues>();
+
+  const handleARemoveReminder = () => {
+    setValue('reminder', null);
+  };
+
+  return (
+    <StyledNotificationWrapper>
+      <StyledLabelLarge sx={{ margin: theme.spacing(1.2, 0, 0, 1.1) }}>
+        {t('reminder')}
+      </StyledLabelLarge>
+      <StyledReminder>
+        <Header onClickHandler={handleARemoveReminder} />
+        <StyledFlexTopCenter>
+          <StyledInputWrapper>
+            <InputController
+              label={t('activityIncomplete')}
+              type="number"
+              name="reminder.activityIncomplete"
+              control={control}
+              InputProps={{ inputProps: { min: 1 } }}
+              endTextAdornmentSingular={t('day')}
+              endTextAdornmentPlural={t('days')}
```

Comment:
I suppose we can use this for plural - https://www.i18next.com/translation-function/plurals#singular-plural

Reply:
you just need to call `t('day', {count: {countDays}})`
and create two translations: 
```
"day_one": "day",
"day_other": "days"
```